stix2slider/convert_cyber_observables.py

```diff
@@ -209,23 +228,36 @@ def convert_pdf_file_extension(pdf_ext, file1x, obs20_id):
                         file1x.metadata.document_information_dictionary,
                         PDF_DOCUMENT_INFORMATION_DICT_MAP,
                         obs20_id)
+    if "pdfid0" in pdf_ext or "pdfid1" in pdf_ext:
+        warn("Order may not be maintained for pdfids in %s", 514, obs20_id)
```

Comment:
Why would the order not be maintained? The order in the STIX 2.0 JSON isn't semantically meaningful, and in combined trailer `pdfid0` (correctly) comes before `pdfid1`. What am I missing?